AmplifyPlugins/Core/AWSPluginsCore/ServiceConfiguration/AmplifyAWSServiceConfiguration.swift

```diff
@@ -0,0 +1,37 @@
+//
+// Copyright 2018-2019 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import AWSCore
+
+public class AmplifyAWSServiceConfiguration: AWSServiceConfiguration {
+    static let baseUserAgent: String = {
+        let version = "0.9.0"
```

Comment:
Let's add a TODO to pick up the version from a central location. @iartemiev is exploring version management as part of their work into Amplify CI/CD.

https://github.com/aws-amplify/amplify-ios/issues/276

Reply:
sounds good, i'll put a comment that references 276